Mathlib/Algebra/Group/Units/Hom.lean

```diff
@@ -218,8 +218,8 @@ variable {G R S T F : Type*}
 
 variable [Monoid R] [Monoid S] [Monoid T] [FunLike F R S]
 
-/-- A local ring homomorphism is a map `f` between monoids such that `a` in the domain
-  is a unit if `f a` is a unit for any `a`. See `IsLocalRing.local_hom_TFAE` for other equivalent
+/-- A map `f` between monoids is *local* if any `a` in the domain
+  such that `f a` is a unit is a unit. See `IsLocalRing.local_hom_TFAE` for other equivalent
```

Comment:
I'm proposing to rephrase this as
```suggestion
/-- A map `f` between monoids is *local* if any `a` in the domain is a unit
  whenever `f a` is a unit. See `IsLocalRing.local_hom_TFAE` for other equivalent
```
to avoid the sentence ending in "is a unit is a unit".